script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -486,8 +489,9 @@ There's something I want to ask you, if that's okay."
 [sha:bf78efc8cad480a7a8371f1c1b2663b18c6f8285]{
 -- Page 331, Offset 25089.
 -- 「あのさ、おまえは吸血鬼だろ？　なら食事って呼べるものは血液だけじゃないのかなって、そう疑問に思った」
+// "Eat" sounds weird for a liquid, but "drink" doesn't quite work in this context
 "So, you're a vampire, right?
- What I was wondering is... well, vampires only eat blood, don't they?"
+ What I was wondering is... well, vampires only feed on blood, don't they?"
```

Comment:
Hmm, feed on feels a bit clinical / predatory here to me for the context of the question (a dumb one). Might be ok with eat here even if it's a bit weird texturally?

Reply:
I prefer "feed on" personally. It's a very vampire-y word according to a documentary I watched called buffy the vampire slayer